gen/rules/src/main/scala/clue/gen/GraphQLGen.scala

```diff
@@ -8,6 +8,7 @@ import scala.meta._
 import cats.syntax.all._
 import metaconfig.Configured
 import cats.effect.IO
+import cats.effect.unsafe.implicits.global
```

Comment:
Why is this import needed?

Reply:
This files makes calls to `unsafeRunSync`. In general we may prefer to use dispatcher and let the runtime call it
But in this case this will run (If I understood correctly) in `scalafix` thus we should call `unsafe` methods directly